### PR TITLE
Drop regexp for net stats parsing

### PR DESCRIPTION
**Pull request by jimmidyson**
Reported in kubernetes/kubernetes#16296

/cc @vishh @yujuhong

### CODE REVIEW

File: container/libcontainer/helpers.go [hunk 1]
```diff
@@ -166,22 +167,16 @@ func scanInterfaceStats(netStatsFile string) ([]info.InterfaceStats, error) {
...
 
-			_, err := fmt.Sscanf(line, "%s %d %d %d %d %d %d %d %d %d %d %d %d %d %d %d %d",
-				&i.Name, &i.RxBytes, &i.RxPackets, &i.RxErrors, &i.RxDropped, &bkt, &bkt, &bkt,
-				&bkt, &i.TxBytes, &i.TxPackets, &i.TxErrors, &i.TxDropped, &bkt, &bkt, &bkt, &bkt)
+		i := info.InterfaceStats{}
 
-			if err != nil {
-				return stats, fmt.Errorf("failure opening %s: %v", netStatsFile, err)
-			}
+		numRead, _ := fmt.Sscanf(line, netstatsLine,
```
Comment (yujuhong) [inline container/libcontainer/helpers.go]: Do we not care about the error anymore?...

Comment (jimmidyson) [inline container/libcontainer/helpers.go]: I don't think so. we've handled the file read error so we have a string that we're just trying to match.

Comment (vishh) [inline container/libcontainer/helpers.go]: Wouldn't Sscanf return an error if the expected number of elements, as specified in the `format`, is not present?

**Comment by jimmidyson** [comment]
Switched to using the error rather than the number of fields read.

**Comment by jimmidyson** [comment]
retest this please

**Comment by jimmidyson** [comment]
retest this please

**Comment by jimmidyson** [comment]
retest this please

**Comment by jimmidyson** [comment]
Tested myself & works fine against coreos beta...

**Comment by yujuhong** [comment]
lgtm. Thanks for the fix!

**Comment by jimmidyson** [comment]
Merging after LGTM.

**Pull request merged**